clouddriver-kubernetes/src/test/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/handler/KubernetesStatefulSetHandlerSpec.groovy

```diff
@@ -0,0 +1,199 @@
+package com.netflix.spinnaker.clouddriver.kubernetes.v2.op.handler
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.description.manifest.KubernetesKind
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.description.manifest.KubernetesManifest
+import org.yaml.snakeyaml.Yaml
+import org.yaml.snakeyaml.constructor.SafeConstructor
+import spock.lang.Specification
+
+
+class KubernetesStatefulSetHandlerSpec extends Specification {
+  def objectMapper = new ObjectMapper()
+  def yaml = new Yaml(new SafeConstructor())
+  def handler = new KubernetesStatefulSetHandler()
+
+  def IMAGE = "gcr.io/project/image"
+  def ACCOUNT = "my-account"
+  def CLUSTER_SIZE = 5
+  def VERSION = "version"
+  def NAMESPACE = "my-namespace"
+  def NAME = "my-name"
+  def KIND = KubernetesKind.STATEFUL_SET
+
+  def statefulSetManifest(Integer partition = CLUSTER_SIZE) {
+
+    return """
```

Comment:
I've always been partial to storing yml snippets and other text fixtures under `src/test/resources` somewhere reasonable and then loading them off the class path, mostly for the sake of keeping big multiline fixtures out of the test code itself

Reply:
Alternatively, I wonder if we can build up a manifest programmatically with `KubernetesManifest`

Reply:
Seems it might be possible https://github.com/DataDog/clouddriver/blob/master/clouddriver-kubernetes/src/test/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/KubernetesManifestAnnotatorSpec.groovy#L30-L35

Reply:
We talked about this IRL and it makes more sense to continue the path we're on, to more closely match the use case of YAML -> KubernetesManifest object, which is closer to the actual code-path users would take. Might move that YAML out into a fixture though.